vendor/photoshop/OpenColorIO_PS_Context.cpp

```diff
@@ -152,7 +152,7 @@ OpenColorIO_PS_Context::getLUTProcessor(OCIO::Interpolation interpolation, OCIO:
     
     OCIO::ConstProcessorRcPtr processor = _config->getProcessor(transform);
     
-    OCIO::ConstCPUProcessorRcPtr cpu_processor = processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_DEFAULT, OCIO::FINALIZATION_EXACT);
+    OCIO::ConstCPUProcessorRcPtr cpu_processor = processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_LOSSLESS);
```

Comment:
Recommend the default (VERY_GOOD).

Reply:
I figured for Photoshop I may as well stick with LOSSLESS because the user is only rendering one frame, but I'm happy to take your recommendation.